app/views/trainees/diversity/ethnic_groups/edit.html.erb

```diff
@@ -0,0 +1,30 @@
+<%= content_for(:breadcrumbs) do %>
+  <%= render_component GovukComponent::BackLink.new(
+    text: "Back",
+    href: trainee_path(@trainee)
+  ) %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds-from-desktop">
+    <%= form_with(model: @ethnic_group, url: trainee_diversity_ethnic_group_path(@trainee), method: :put, local: true) do |f| %>
+      <%= f.govuk_error_summary %>
+
+        <%= f.govuk_collection_radio_buttons(
+          :ethnic_group,
+          format_ethnic_group_options(Trainee.ethnic_groups.keys),
+          -> (value) { value },
+          -> (name) { I18n.t("views.ethnic_groups.names.#{name}") },
+          -> (description) { I18n.t("views.ethnic_groups.descriptions.#{description}") },
+          form_group: { classes: "govuk-!-margin-bottom-0" },
+          legend: { text: "What is their ethnic group?", tag: "h1", size: "l" }) %>
+
+        <div class="govuk-form-group">
+          <%= f.govuk_radio_divider %>
+          <%= f.govuk_radio_button :ethnic_group, :no_ethnicity_provided, label: { text: "Not provided" } %>
+        </div>
```

Comment:
@edwardhorsford That is a very good point. We will need to fix that because if we leave it the `Not provided` will cause an accessibility issue because it does have any context for the option. The fieldset above it has the page heading set.

I think there is a way around it. Wrap another a fieldset around both groups of radio buttons. I'm not sure about the first group of radio buttons legend's copy but something along those lines'. 
```suggestion
      <%= f.govuk_fieldset(legend: { text: "What is their ethnic group?", tag: "h1", size: "l" }) do %>
        <%= f.govuk_collection_radio_buttons(
          :ethnic_group,
          format_ethnic_group_options(Trainee.ethnic_groups.keys),
          -> (value) { value },
          -> (name) { I18n.t("views.ethnic_groups.names.#{name}") },
          -> (description) { I18n.t("views.ethnic_groups.descriptions.#{description}") },
          form_group: { classes: "govuk-!-margin-bottom-0" },
          legend: { text: "Select which ethnic group", hidden: true }) %>

        <div class="govuk-form-group">
          <%= f.govuk_radio_divider %>
          <%= f.govuk_radio_button :ethnic_group, :no_ethnicity_provided, label: { text: "Not provided" } %>
        </div>

      <% end %>
```

Reply:
Unfortunately, I don't think its possible to suppress `govuk_collection_radio_buttons` fieldset so there has to be a fieldset within a fieldset.

Reply:
Double fieldsets wouldn't be good. I think the formbuilder can do this if you do the radio options individually rather than with `govuk_collection_radio_buttons`. Otherwise I think the best bet would be to do the markup manually.

Reply:
@darokel asked me about this on Slack yesterday afternoon, I pointed him in the way of [#govuk_radio_buttons_fieldset](https://govuk-form-builder.netlify.app/form-elements/radios/#radio-buttons-fieldset-with-conditional-content-and-a-divider) which I believe will do what you want.

Reply:
Yup it will. I'm making the change now.

Reply:
@edwardhorsford @aliuk2012 Have fixed this in https://github.com/DFE-Digital/register-trainee-teachers/pull/109/commits/3ed89eeb77786dd7d8086d93218abc7aa45083f3. Review app should be updated in a bit. Have updated screenshots

Reply:
Thank you @peteryates for the tip 👍